crates/lib/src/qpu/api.rs

```diff
@@ -94,15 +95,69 @@ pub async fn submit(
     client: &Qcs,
     execution_options: &ExecutionOptions,
 ) -> Result<JobId, QpuApiError> {
+    submit_with_parameter_batch(
+        quantum_processor_id,
+        program,
+        std::iter::once(patch_values),
+        client,
+        execution_options,
+    )
+    .await?
+    .pop()
+    .ok_or_else(|| GrpcClientError::ResponseEmpty("Job Execution ID".into()))
+    .map_err(QpuApiError::from)
+}
+
+/// Execute a compiled program on a QPU with multiple sets of `patch_values`.
+///
+/// This action is *atomic* in that all jobs will be queued, or none of them will. On success, this
+/// function will return a vector of [`JobId`] where the length and order correspond to the
+/// `patch_values` given. However, note that execution in the order of given patch values is not
+/// guaranteed. If there is a failure to queue any of the jobs, then none will be queued.
+///
```

Comment:
Yeah, I had a similar thought. I admit that the API client makes more sense as a source of truth. Still, since the docs users see is dependent on their client version, they can still see an old docstring even if we are vigilant about keeping the clients version up to date.